balances-adjustment/src/main/java/com/powsybl/balances_adjustment/balance_computation/BalanceComputationFactoryImpl.java

```diff
@@ -20,8 +20,29 @@
  */
 public class BalanceComputationFactoryImpl implements BalanceComputationFactory {
 
+    private final boolean isStatic;
+
+    public BalanceComputationFactoryImpl() {
+        this(false);
+    }
+
+    public BalanceComputationFactoryImpl(boolean isStatic) {
+        this.isStatic = isStatic;
```

Comment:
Is static the vocabulary commonly used when we ignore the loadflow or is there a better word?

Reply:
It's not commonly used, probably there is a better word. It means it is based on "static" fields of the network elements generators(targetP) and loads(P0) and not on "dynamic" fields computed by the loadflow ie P. Do you have a suggestion ?

Reply:
maybe something like "useTargetInjections"
This might also help make it more explicit how the boolean changes the computation